functions.php

```diff
@@ -617,3 +617,36 @@ function ($theme_json) {
         return $theme_json->update_with($new_color_palette);
     }
 );
+
+
+add_action('admin_init', function (): void {
+    register_setting(
+        'discussion',
+        'gdpr_checkbox',
+        [
+            'type' => 'boolean',
+            'default' => planet4_get_option('gdpr_checkbox'),
```

Comment:
This function will search the `gdpr_checkbox` value in '[planet4_options](https://github.com/greenpeace/planet4-master-theme/blob/main/functions.php#L77)' key data.

Reply:
Yes the idea was to use that "old" value as default for the new setting, but I see Flo added a migration instead which is probably better